web/src/components/workflow/component/stage/StageField.js

```diff
@@ -82,14 +82,14 @@ class StageField extends React.Component {
                       {/* // TODO: 暂时不展示此字段 */}
                       {/* <Field
                         label={'ENTRYPOINT'}
-                        name={`${specKey}.spec.containers.${index}.command`}
+                        name={`${specKey}.spec.containers.${index}.args`}
```

Comment:
Here it should be `command` for `ENTRYPOINT`. Please remove the this part as not used.